stdlib/REPL/test/precompilation.jl

```diff
@@ -0,0 +1,37 @@
+
+## Tests that compilation in the interactive session startup are as expected
+
+f, _ = mktemp()
+cmd = `$(Base.julia_cmd()[1]) --trace-compile=$f -q --startup-file=no -e "Timer((t)->exit(), 5)" -i`
+@test success(pipeline(cmd, stdin=stdin))
+out = read(f, String)
+@debug "trace compile output:\n$out"
+
+expected_precompiles = if Sys.isapple()
+    1
```

Comment:
```suggestion
    4
```
@KristofferC locally I see 661.. maybe pkgimages are somehow not multiversioning properly?

Reply:
Sorry, scratch that.. I see 661 with a local build.. weird

Reply:
I guess the test io needed to be set up better